OloEditor/SandboxProject/Assets/Scripts/LuaScripts/SteamPlatformDemo.lua

```
@@ -0,0 +1,111 @@
+-- do nothing. Check isAvailable() only to hide UI or skip work you would rather not do — never as
+-- a safety guard.
+--
+-- To see any of this actually happen you need the Steam client running and a `steam_appid.txt`
+-- containing `480` (Valve's public Spacewar test app, which every Steam account owns) in the
+-- working directory — OloEditor/. See docs/ops/build.md § "Steamworks SDK".
+--
+-- The achievement ids below are Spacewar's real, published ones, so this script unlocks
+-- genuinely against App ID 480. Replace them with your own game's ids.
+
+local ACH_WIN_ONE_GAME = "ACH_WIN_ONE_GAME"
+local ACH_TRAVEL_FAR_ACCUM = "ACH_TRAVEL_FAR_ACCUM"
```

Luacheck (1.2.0)

[warning] unused variable 'ACH_TRAVEL_FAR_ACCUM'

(W211)